src/bench/base58.cpp

```diff
@@ -13,7 +13,7 @@
 
 static void Base58Encode(benchmark::State& state)
 {
-    unsigned char buff[32] = {
```

Comment:
It's perfectly legal to use a pointer that points one past the end of an object (and the function does). You can't dereference it, though.